src/components/PersonLink.tsx

```diff
@@ -0,0 +1,54 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={person.slug}
+          className={classNames({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {person.mother ? (
+          <Link
+            to={`/people/${person.mother.slug}`}
+            className="has-text-danger"
+          >
+            {person.mother.name}
+          </Link>
```

Comment:
<img width="672" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/86789161/44c9f72d-e57e-4f68-90c6-ec6b88e6939d">

Create separate component for this and reuse it in all places